app/scripts/SinkhornNNLSLinseedC.R

```diff
@@ -624,8 +636,14 @@ SinkhornNNLSLinseed <- R6Class(
       self$init_X <- cbind(1/sqrt(self$N),
                               rbind(self$init_X,
                                     -1/(self$cell_types-1) * apply(self$init_X,1,sum)))
+      self$setInitParamsByX()
+    },
+
+    setInitParamsByX = function() {
```

Comment:
However the basis generated from this function doesn't seem to be good in 10d:
<img width="872" alt="image" src="https://github.com/aladyeva-wustl/docker_linseed_snakemake/assets/36701344/60612c8b-8a09-40a1-a5aa-3111f29fe60e">

Here's the math for it:
<img width="1017" alt="image" src="https://github.com/aladyeva-wustl/docker_linseed_snakemake/assets/36701344/dfd06d40-0114-40b2-9630-56a5be1587eb">